src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy

```diff
@@ -1,96 +1,96 @@
 package org.openkilda.functionaltests.spec.server42
 
 import static org.junit.Assume.assumeTrue
-import static org.openkilda.testing.Constants.STATS_LOGGING_TIMEOUT
+import static org.openkilda.testing.Constants.WAIT_OFFSET
 
 import org.openkilda.functionaltests.HealthCheckSpecification
+import org.openkilda.functionaltests.extension.failfast.Tidy
+import org.openkilda.functionaltests.helpers.SwitchHelper
 import org.openkilda.functionaltests.helpers.Wrappers
 import org.openkilda.messaging.model.system.FeatureTogglesDto
-import org.openkilda.model.FlowEncapsulationType
-import org.openkilda.northbound.dto.v1.switches.SwitchPropertiesDto
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch
 
 import groovy.time.TimeCategory
 import org.springframework.beans.factory.annotation.Value
-import spock.lang.Ignore
 import spock.lang.Narrative
 import spock.lang.Shared
 import spock.util.mop.Use
 
 import java.util.concurrent.TimeUnit
 
-@Ignore("unstable on jenkins, fix ASAP")
 @Use(TimeCategory)
 @Narrative("Verify that statistic is collected from server42 Rtt")
+/* On local environment these tests will use stubs without sending real rtt packets across the network.
+see server42-control-server-stub
+ */
 class Server42RttSpec extends HealthCheckSpecification {
     @Shared
     @Value('${opentsdb.metric.prefix}')
     String metricPrefix
 
-    def "Create two flow with server42 Rtt feature and check datapoints in opentsdb"() {
-        given: "Two active neighboring switches one with server42 and one without"
+    @Tidy
+    def "Flow rtt stats are available in forward and reverse directions for new flows"() {
```

Comment:
what do you mean `for new flows`? Flow rtt stats are not available for a flow that is created before enabling server42FlowRtt , am I right?

Reply:
This text is not implying that it is not available for old flows, it is only stating what is checked in the test - rtt is available for new flows (aka flows created after enabling toggles)